### DISCUSSION TITLE
Set screen boundaries?

### DISCUSSION THREAD

**Discussion by 4194304** [i3/i3#5935] [Q&A]
I don't know if this can be fixed or not, but I have my display set to 1920x1200. It's a custom resolution I made with xrandr, so I wouldn't have to deal with DPI scaling issues or neck pain. The problem is that i3 uses the full 2880x1800 screen space when it really shouldn't. Tiling mode is broken and is half cut off, and things like tint2 and i3bar just disappear. If there is a way to set screen boundaries so I don't have to deal with that, it would save me quite a bit of time.

(At native resolution 2880x1800 everything shows like it should)

**Comment by okraits** [comment]
How do you set the reduced resolution? What is the output of `xrandr -q`?
